### DISCUSSION TITLE
Coral TPU Advice

### DISCUSSION THREAD

**Discussion by stamandr** [blakeblackshear/frigate#2032] [Ask A Question]
The Google Coral USB Accelerator is not supported in Canada by the manufacturer , so no retailer can sell it to customers in Canada. That being said, I'm looking for recommendations that have been working well for people using the M.2 Accelerator B+M key or M.2 Accelerator A+E key on a Raspberry PI4 .

Also, what external USB 3 case are you using on the Raspberry PI4 with either the M.2 Accelerator B+M key or M.2 Accelerator A+E key ?

**Comment by ferbulous** [comment]
No external usb 3 case works for any of the m.2 keys and the same goes for compute module pi4 due  to pcie h/w limitation 
You might need to consider nano jetson or forwarding service to get the usb coral to canada

**Reply by stamandr** [reply]
My understanding is the nano jetson doesn't work with Frigate. Have you tried it ?

**Reply by ferbulous** [reply]
Sorry, I assumed it work, I have a friend who owns one but he's only using for doods & deepstack. I'll try to ask if he got it working
